SPECS/ypbind/ypbind.spec

```diff
@@ -113,8 +112,39 @@ install -m 755 %{SOURCE5} $RPM_BUILD_ROOT%{_libexecdir}/ypbind-post-waitbind
 %license COPYING
 
 %changelog
-* Fri Oct 15 2021 Pawel Winogrodzki <pawelwi@microsoft.com> - 3:2.6.1-3
-- Initial CBL-Mariner import from Fedora 32 (license: MIT).
+* Wed Aug 24 2022 Zhichun Wan <zhichunwan@microsoft.com> - 3:2.7.2-10
```

Comment:
That's something I forgot to mention about taking specs from other repos: we remove the `Epoch` tag, because it's unnecessary if this is the first version of the package in Mariner. I think you already removed the tag itself and one thing that's still left, is to remove the `3:` part from your changelog entry (older ones from Fedora are fine the way they are now). You can also mention in the changelog, that you remove the epoch, so that it's clear it wasn't accidental.

Reply:
Done